include/RAJA/pattern/forall.hpp

```diff
@@ -103,7 +103,8 @@
 #include <iterator>
 #include <type_traits>
 
-#include "RAJA/internal/rangelist_forall.hpp"
+#include "RAJA/policy/cuda/forward_decl_cuda.hpp"
```

Comment:
I'd like to find another workaround for this -- the forward declaration of cuda in the generic forall pattern doesn't rest well with me. It should be agnostic to any execution policy